### DISCUSSION TITLE
2009/02/25/%E4%B9%A6%E5%90%8D%E7%A1%AE%E5%AE%9A%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#254] [Announcements]
# 2009/02/25/%E4%B9%A6%E5%90%8D%E7%A1%AE%E5%AE%9A%E4%BA%86

我除了在博客上邀请大家给书名提意见，还在论坛上也发了一个贴：http://www.vihome.com.cn/bbs/viewthread.php?tid=13340&extra=page%3D1

https://blog.qizhen.xyz/2009/02/25/%E4%B9%A6%E5%90%8D%E7%A1%AE%E5%AE%9A%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

lei
2009年04月24日 09:15 编辑
恭喜恭喜，这个名字听起来非常舒服！ 令人期待的内容！

回复
Yigang
2009年02月27日 09:23 编辑
我夏天回国，要是出了书我一定买一本带过来，要是有签名最好啦，呵呵

回复
Jun
2009年02月26日 16:30 编辑
恭喜！什么时候出版呀？

回复
Nan
2009年02月26日 09:07 编辑
恭喜呀！

回复
俊
2009年02月26日 09:03 编辑
恭喜恭喜，记得签名寄给偶一份哈，严重期待ing

回复
Haiying
2009年02月25日 17:19 编辑
这个名字很好，简单明了。等待签名售书~~

回复
wen
2009年02月25日 13:24 编辑
congratulation!!!

回复
帆
2009年02月25日 12:51 编辑
恭喜啊，期待大作问世

回复
